x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/logging/DeprecationCacheResetAction.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.deprecation.logging;
+
+import org.elasticsearch.action.ActionRequest;
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.action.ActionResponse;
+import org.elasticsearch.action.ActionType;
+import org.elasticsearch.common.io.stream.StreamInput;
+
+import java.io.IOException;
+
+/**
+    Resets deprecation indexing rate limiting cache on each node.
+ */
+public class DeprecationCacheResetAction extends ActionType<ActionResponse.Empty> {
```

Comment:
I wasn't sure I extended the right classes in DeprecationCacheResetAction and TransportDeprecationCacheResetAction
My goal was to keep it as simple as possible. 
Is there a generic "empty request" somewhere in our codebase already?
Also do we have a way to make sure that a response (even empty) is returned to a rest client only when operations on every node succeeded?